pkg/types/baremetal/platform.go

```diff
@@ -34,6 +34,7 @@ type Host struct {
 	HardwareProfile string           `json:"hardwareProfile"`
 	RootDeviceHints *RootDeviceHints `json:"rootDeviceHints,omitempty"`
 	BootMode        BootMode         `json:"bootMode,omitempty"`
+	NetworkConfig   string           `json:"networkConfig,omitempty"`
```

Comment:
Are we sure that we want this to be a `string` rather than a `map[string]interface{}`? I would expect the user to want to enter in-line yaml rather than a text string.

Reply:
The installer manages it essentially as a blob (it will be stored also in a secret, see PR [5247](https://github.com/openshift/installer/pull/5247)) and will pass it as it is to the image builder container (with a just a minimal validation).